WireAPI/Sources/WireAPI/APIs/ConversationsAPI/ConversationsAPIBuilder.swift

```diff
@@ -37,19 +39,20 @@ public struct ConversationsAPIBuilder: APIBuilder {
     public func makeAPI(for version: APIVersion) -> any ConversationsAPI {
         switch version {
         case .v0:
-            ConversationsAPIV0(httpClient: httpClient)
+            ConversationsAPIV0(httpClient: httpClient, backendDomain: backendDomain)
         case .v1:
-            ConversationsAPIV1(httpClient: httpClient)
+            // TODO: we only need backend info for V0, but now pass to all?
```

Comment:
This is due to the inheritance of classes. Any ideas how to avoid it and make it clear, that it can be refactored simply in future?